tests/test_auth.py

```diff
@@ -22,6 +22,11 @@
 )
 
 
+# FIXME: itsdangerous.py has EPOCH hardcoded to 2011/01/01 for whatever reason,
+#        which breaks when using dates earlier than that.
+#        Fix has been implemented, awaiting next release:
+#        https://github.com/pallets/itsdangerous/issues/46
+@patch('itsdangerous.EPOCH', 0)
```

Comment:
Erm, does the test really rely on a date that early?

Reply:
They did, `@freezegun.freeze_time('2001-01-01')`
However, I can't seem to reproduce the error now. Weird.
I'll remove it and see how it goes.

Reply:
My point was rather that the date likely needn't be _that_ earlier — hopefully `20110102` would suffice?